### ISSUE TITLE
Fix the styling (for example font, color, and bg color) to match the standard IntelliJ console styling

### ISSUE THREAD

**Issue by chairbender** [chairbender/intellij-object-calisthenics-analyzer#4]
For example - the background should match the background of the console window or events log. Right now it matches the background of windows that don't have console-style elements, like the package explorer.

Additionally, the current code link color, when using the Darcula theme, makes the links basically undreadable. Use the same color that the console and event log window uses for clickable text.

Basically, ensure that all values are coming from the current IntelliJ theme so it looks like just another console window or event viewer. Make sure the font style, font size, and color are based on the intellij settings, if possible.

**Comment by chairbender** [comment]
fixed to use the defaults for the current theme. Took a lot longer than it should've, this API really lacks solid documentation...
